pkg/api/config.go

```diff
@@ -74,6 +74,10 @@ type ImageConfig struct {
 	// used in the cluster.
 	Format string `json:"format,omitempty"`
 
+	// Image names with end "Base" are being use by operators.
```

Comment:
why continue to append Base to the names?

Reply:
Because these are "base" images based on Operators naming convention. We could split them to separate section. It is good to have some kind naming distinction so just from first look we would know that these are not standard images we use, but with operators management layer.  

Happy to add some other distinguishment. This is really just esthetics for code reading

Reply:
Agreed to remove Base then lgtm